frontend/views/pages/PayGroup.vue

```diff
@@ -1,122 +1,128 @@
 <template lang="pug">
-page(pageTestName='payGroupPage' pageTestHeaderName='payGroupTitle')
-  template(#title='') Pay Group
-
-  template(#sidebar='')
-    h2.subtitle
-      i18n Payments Sent
-
-    h3
-      i18n(
-        :args='{ sent: paymentSummary.sent, total: fakeStore.users.length }'
-      ) {sent} of {total}&nbsp;
-
-      tooltip(v-if='paymentSummary.hasWarning')
-        i.icon-exclamation-triangle
-
-        template(slot='tooltip')
-          strong Payment Declined
-          i18n(tag='p')
-            | Someone didn&rsquo;t confirm your payment. Please mark as payed only when it&rsquo;s done.
-
-    h2.subtitle
-      i18n Payments Confirmed
-
-    h3
-      i18n(:args='{ sent: paymentSummary.confirmed, total: fakeStore.users.length }')
-        | {sent} of {total}
-
-    h2.subtitle
-      i18n Amount Sent
-
-    h3(:class="{'has-text-success': paymentAllDone}")
-      i18n(
-        v-if='paymentAllDone'
-        :args='{ currency: fakeStore.currency, amountTotal: paymentSummary.amountTotal }'
-      ) All {currency}{amountTotal}
-
-      i18n(
-        v-else=''
-        :args='{ currency: fakeStore.currency, amoutPayed:paymentSummary.amoutPayed, amountTotal: paymentSummary.amountTotal }'
-      ) {currency}{amoutPayed} of {currency}{amountTotal}
-
-      progress-bar(
-        :primary='paymentProgress.sent'
-        :secondary='paymentProgress.confirmed'
+page(
+  pageTestName='payGroupPage'
+  pageTestHeaderName='payGroupTitle'
+)
+  template(#title='') {{ L('Pay Group') }}
+
+  template(#sidebar=''
+    v-if='hasPayments'
+  )
+    .c-summary-item(
+      v-for='(item, index) in paymentSummary'
+      :key='index'
+    )
+      h4.title.is-4 {{item.title}}
+      progress-bar.c-progress(
+        :max='item.max'
+        :value='item.value'
+        :hasMarks='item.hasMarks'
       )
-    p
-      i18n What's this page about, so the user understands the context.
-
-  .card
-    table.c-table
-      thead.sr-only
-        tr
-          i18n(tag='th') Name
-          i18n(tag='th') Payment Status
-          i18n(tag='th') Amount
-
-      tfoot
-        tr
-          i18n.c-table-cell(tag='th' colspan='2')
-            | Total
-          td
-            | {{fakeStore.currency}}{{paymentSummary.amountTotal}}
-
-      tbody
-        tr(
-          v-for='(user, index) in fakeStore.users'
+      p(:class="{'has-text-success': item.max === item.value}")
+        i.icon-check(v-if="item.max === item.value")
+        .has-text-1 {{item.label}}
```

Comment:
Before you had this as `i18n.has-text-1`, which set this as a `span` by default, so since you fixed this, I think now this will be a `div`. Did you want it to be a `span`? If so this should be `span.has-text-1`

Reply:
It's whatever in this case, both do the job :)